talk/talk-api.yaml

```
@@ -2286,6 +2286,13 @@ paths:
           schema:
             type: integer
             format: int64
+        - name: timeZoneOffset
+          in: query
+          required: true
+          description: The difference in minutes detween local time and GMT.
```

```suggestion
          description: The difference in minutes between local time and UTC.
```
GMT is a slightly different term

---

I think we need to make the same change in reports.